seis/dl/doc.go

```diff
@@ -0,0 +1,5 @@
+// The dl module has been writen as a lightweight replacement for the C libdali library.
+// It is aimed at clients that need to connect to a datalink server, either requesting inforamtion
+// or for uploading most likely miniseed records.
+//
```

Comment:
Is this file to be further completed?

Reply:
in terms of more documentation, or whether there's actually any code?  Let me know if there's anything missing if its more docs, otherwise see  https://blog.golang.org/godoc but they do mentions `doc.go` rather this `docs.go`.